### ISSUE TITLE
Public folder symbolic icon doesn't have enough contrast in dark style

### ISSUE THREAD

**Issue by JoseExposito** [elementary/icons#985]
## What Happened

This icon: https://github.com/elementary/icons/blob/master/places/symbolic/folder-publicshare-symbolic.svg
Does not have enough contrast in dark style.

![Public icon bug](https://user-images.githubusercontent.com/1335948/103441845-1dbe3a00-4c51-11eb-9aff-a07b111b852d.png)

## Expected Behavior

The icon should have more contrast or the folder version should be used when importing images to photos.

## Steps to Reproduce

1. Open the Photos app
2. Click on Settings -> Import from folder
3. The public folder icon doesn't have enough contrast

## Platform Information

elementary OS 6

**Comment by lainsce** [comment]
Check out #938 I think it's the icon that is used here, but adapted to be more visible in dark mode.

**Comment by JoseExposito** [comment]
Yes, it could be that icon. Let me know when your PR is merged so I can test it and close this issue :+1:
